morpheus/_lib/cmake/libmorpheus.cmake

```diff
@@ -92,7 +92,7 @@ add_library(${PROJECT_NAME}::morpheus ALIAS morpheus)
 target_link_libraries(morpheus
   PRIVATE
     matx::matx
-    ZLIB::ZLIB
+    $<$<CONFIG:Debug>:ZLIB::ZLIB>
```

Comment:
Should `tritonclient` just set this as `PUBLIC` so we dont need to include this on our own? I dont believe that we use Zlib directly

Reply:
Hmm. If we're not using zlib directly, then making it public on triton client for debug builds seems reasonable. I'd make an issue for it and follow up later since it requires changes to the utilities repo as well.

Reply:
https://github.com/nv-morpheus/Morpheus/issues/1495